src/convolution.cpp

```diff
@@ -163,6 +163,12 @@ ConvolutionDescriptor::GetForwardOutputTensorWithLayout(const TensorDescriptor&
 
     auto wei_spatial = boost::adaptors::slice(wDesc.GetLengths(), 2, 2 + spatial_dim);
 
+    if(wDesc.GetLayout_t() == miopenTensorCHWNc4 || wDesc.GetLayout_t() == miopenTensorCHWNc8)
```

Comment:
```suggestion
    if(wDesc.IsVectorized())
```

Reply:
Actually, we can not only IsVectorized or not here to determine the following code, since weight tensor is not in default NCHW.